ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadTest.java

```diff
@@ -225,10 +240,10 @@ public void shouldRespondWithSyncingDuringBackwardsSync() {
     BlockHeader mockHeader = new BlockHeaderTestFixture().buildHeader();
     var resp = resp(mockPayload(mockHeader, Collections.emptyList()));
 
-    EngineExecutionResult res = fromSuccessResp(resp);
+    EnginePayloadStatusResult res = fromSuccessResp(resp);
```

Comment:
we need the backward syncing mock, and also there is a peculiarity about baseFee since it is optional - you have to specify it for the hash to match

```suggestion
    when(mergeCoordinator.latestValidAncestorDescendsFromTerminal(any(BlockHeader.class)))
        .thenReturn(true);
    when(mergeCoordinator.isBackwardSyncing())
        .thenReturn(Boolean.TRUE);
    BlockHeader mockHeader = new BlockHeaderTestFixture().baseFeePerGas(Wei.ONE).buildHeader();
```

Reply:
the mock for isBackwardSyncing would change to be for ifBackwardSyncing(...) if/when you implement the mergeCoordinator suggestion

Reply:
I've adapted the test now